modules/nf-core/metaspace/submit/tests/nextflow.config

```
@@ -0,0 +1,11 @@
+env {
+    // NOTE This is how nf-core/metaspace/submit users will use METASPACE submission in real world use
+    API_KEY = "$METASPACE_API_KEY"
+
+    // NOTE This is how nf-core/metaspace/submit users will test out METASPACE submission with a user API key
+    // nextflow secrets set METASPACE_API_KEY "USER_API_KEY"
```

? Which of these would be the real use case?

---

I get the confusion. I think secrets has to be defined by the user using `nextflow secrets set METASPACE_API_KEY "USER_API_KEY"`. I was trying to mimic the setup from sentieon modules (https://github.com/nf-core/modules/blob/9fe7867a4f012db581d7bd560228ded1074eb0e5/modules/nf-core/sentieon/dedup/tests/nextflow.config). 

So it can be tested locally without the need to use `nextflow secrets` by just export api key as environment variable. Or it could be set by `nextflow secrets`. Both should work.

---

```suggestion
    
    API_KEY = "$METASPACE_API_KEY"

    // NOTE This is how nf-core/metaspace/submit users will set their user API key locally
    // nextflow secrets set METASPACE_API_KEY "USER_API_KEY"
```
@SPPearce Is this more clear ?